File: Angular-InterviewScheduler/src/app/models/recruiter.ts
import { Time } from "@angular/common";

export class Recruiter {
    recruiterId :number=0;
    date? :Date;
    time?:string;
    rounds?:number;
    name?:string;
    designation?:string;
    emailId?:string;
    interviewStatus?:string;
    recomendedDesignation?:string;
    remarks?:string;
    offerLetterStatus ?:string;
    candidateId?:number;
constructor(recruiterId :number=0,
    date :Date,
    time:Time,
    rounds:number,
    name:string,
    designation:string,
    emailId:string,
    interviewStatus:string,
    recomendedDesignation:string,
    remarks:string,
    offerLetterStatus :string,
    candidateId:number){
        this.recruiterId=recruiterId;
        this.date=date;
        this.time=this.formatTime(time);
        this.rounds=rounds;
        this.name=name;
        this.designation=designation;
        this.emailId=emailId;
        this.interviewStatus=interviewStatus;
        this.recomendedDesignation=recomendedDesignation;
        this.remarks=remarks;
        this.offerLetterStatus=offerLetterStatus;
        this.candidateId=candidateId;
}
private formatTime(time: Time): string {
    // Convert Time to ISO 8601 string format
    const hours = time.hours.toString().padStart(2, '0');
    const minutes = time.minutes.toString().padStart(2, '0');
    return `${hours}:${minutes}:00`;
}
}

File: InterviewScheduler_API/InterviewScheduler/Model/Recruiter.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore.Metadata.Internal;

namespace InterviewScheduler.Model
{
    [Table("recruiters")]
    public class Recruiter
    {
        [Key]
        [Column("recruiterid")]
        public int RecruiterId {  get; set; }
        [Column("date")]
        public DateOnly Date {  get; set; }
        [Column("time")]
        public TimeOnly Time { get; set; }
        [Column("rounds")]
        public int Rounds { get; set; }
        [Column("name")]
        public string Name {  get; set; }
        [Column("designation")]
        public string Designation { get; set; }
        [Column("emailid")]
        public string EmailId {  get; set; }
        [Column("status")]
        public string InterviewStatus {  get; set; }
        [Column("recomendeddesignation")]
        public string RecomendedDesignation {  get; set; }
        [Column("remarks")]
        public string Remarks { get; set; }
        [Column("offerletterstatus")]
        public string offerLetterStatus { get; set; }

        [Column("candidateid")]
        [ForeignKey("CandidateId")]
        public int CandidateId {  get; set; }

        //public Candidates candidate { get; set; }



    }
}
